Source/Shaders/Builtin/Constants/ellipsoidRadii.glsl

```diff
@@ -0,0 +1,7 @@
+/**
+ * 0.1
+ *
+ * @name czm_ellipsoidRadii
```

Comment:
Same comment as in the inverse radii constant, maybe it should say something like:

> The radius of the WGS84 ellipsoid.